content/docs/2.4/concepts/scaling-jobs.md

```diff
@@ -120,13 +120,35 @@ scalingStrategy:
 
 Select a Scaling Strategy. Possible values are `default`, `custom`, or `accurate`. The default value is `default`.
 
+> 💡 **NOTE:**
+>
+>`maxScale` is not the running Job count. It is measured through:
+ >```go
+ >maxValue = min(scaledJob.MaxReplicaCount(), devideWithCeil(queueLength, targetAverageValue))
+ >```
+ >that means, pure the value queueLength devided by targetAvarageValue unless it is exceed the MaxReplicaCount.
```

Comment:
Did you mean this?
```suggestion
 >That means it will use the value of `queueLength` divided by `targetAvarageValue` unless it is exceeding the `MaxReplicaCount`.
```